resources/page/taxonomy.go

```diff
@@ -112,6 +112,19 @@ func (i Taxonomy) ByCount() OrderedTaxonomy {
 	return ia
 }
 
+// Page returns the taxonomy page or nil if the taxonomy has no terms.
+func (i Taxonomy) Page() Page {
+	var first WeightedPages
+	for _, v := range i {
+		first = v
+		break
+	}
+	if first == nil {
+		return nil
+	}
+	return first.Page().Parent()
```

Comment:
I know I was the one who suggested the above construct, but looking at it now, I suspect it would be simpler to reason about if it looked like:

```go
func (i Taxonomy) Page() Page {
  for _, v := range i {
      return v.Page().Parent()
  }
  return nil
}
```

Reply:
I should have seen that. Thanks.